code_blocks/🚀 Getting Started/installation/ionic_react.ts

```diff
@@ -1,7 +1,7 @@
-import { Purchases } from '@awesome-cordova-plugins/purchases';
+import Purchases, { LOG_LEVEL } from 'cordova-plugin-purchases/www/plugin';
```

Comment:
Maybe this works, but shouldn't this be 
```suggestion
import { Purchases, LOG_LEVEL } from 'cordova-plugin-purchases/www/plugin';
```

Reply:
We are doing `export default Purchases` at the end of the plugin, so we do `import Purchases` to import the default export. If we were not default exporting anything, we would use the second form:

<img width="773" alt="Screenshot 2023-08-23 at 10 57 12 AM" src="https://github.com/RevenueCat/revenuecat-docs/assets/664544/3c81e559-ad0f-4f00-85d9-4c0ea590315e">